interview-questions.md

```diff
@@ -9,59 +9,67 @@ Then, research the question to expand on your answer. Even if you feel you have
 
 1. What is `this` in JavaScript?
 
-  Your answer:
+  Your answer: The `this` command is used to call methods that are in the same string. Instead of calling to the class and method, it allows coders to shortcut the code.
 
-  Researched answer:
+  Researched answer: `this` refers to an object and depending on what type of objects it is, changes how `this` keyword is called.
 
 
 
 2. What is React? Why would you use it?
 
-  Your answer:
+  Your answer: React is a application created by Facebook to handle a large amount of users and their user interface. The reason you would use it is it's ability to quickly refresh a page when new inputs are added by other users.
 
-  Researched answer:
+  Researched answer: React is a JavaScript library which is used for user interfaces on web pages or mobile apps.
 
 
 
 3. Which lifecycle method is required in a React class component?
 
-  Your answer:
+  Your answer: The constructor is the lifecycle method that is required. It allows the parents classes to pass data to the child classes without having to make the same information in the child classes.
 
-  Researched answer:
+  Researched answer: React has three lifecyle methods. Mounting, updating, and unmounting which help show and update what is shown on the web page through the DOM.
 
 
 
 4. What is JSX? What is one notable difference between HTML and JSX?
 
-  Your answer:
+  Your answer: JSX is what React uses to make HTML tag codes in a JavaScript file. A notable difference between the two is you can't use JavaScript codes in a HTML file, but in JSX you can use both with some exceptions.
 
-  Researched answer:
+  Researched answer: JSX is a JavaScript extension which allows you to write JavaScript objects into HTML elements.
 
 
 
 5. What is yarn? What file(s) are modified in a React application when you run yarn?
 
-  Your answer:
+  Your answer: Yarn is a file manager which allows you to download specfic files that you need to assist you with making code. I'm not sure what those files are, but I do know when you run yarn with React it will open a window in your browser and displays the code in the React app.
 
-  Researched answer:
+  Researched answer: Yarn is a package manager
 
 
 
 6. STRETCH: What is an anonymous function in JavaScript?
 
-  Your answer:
+  Your answer: I don't know what that is, but from what it sounds like the anonymous function might be a hidden function in JavaScript that only works under certain circumstances.
 
-  Researched answer:
+  Researched answer: A function that hasn't been named.
```

Comment:
Excellent work in this section Jonathan!  When possible, try to expand as much as you can with your researched answer.